elasticsearch/ci-env.sh

```diff
@@ -4,14 +4,19 @@ set -o xtrace
 
 PROMETHEUS_EXPORTER_URL=${PROMETHEUS_EXPORTER_URL:-$MAVEN_REPO_URL/org/elasticsearch/plugin/prometheus/prometheus-exporter/$PROMETHEUS_EXPORTER_VER/prometheus-exporter-$PROMETHEUS_EXPORTER_VER.zip}
 OPENDISTRO_URL=${OPENDISTRO_URL:-$MAVEN_REPO_URL/com/amazon/opendistroforelasticsearch/opendistro_security/$OPENDISTRO_VER/opendistro_security-$OPENDISTRO_VER.zip}
+INGEST_PLUGIN_URL=${INGEST_PLUGIN_URL:-$MAVEN_REPO_URL/org/elasticsearch/plugin/ingest/openshift/$INGEST_PLUGIN_VER/openshift_ingest-plugin-$INGEST_PLUGIN_VER.zip}
```

Comment:
If we never intend to pubblish this to an official maven repo then this line should simply default to  use the URL manipulation that you are doing for CI and/or pulling it straight from the Viaq repo

Reply:
This plugin is not generalized enough atm. It makes not sense to push it into any public mvn repo.

Reply:
correct.  that is exactly my point.  we never intend to publish to maven so this change is incorrect (e.g. $MAVEN_REPO_URL)